scripts/convert-lanes-json-to-flatbuffer.py

```diff
@@ -406,7 +478,12 @@ def check_lane_spine(laneSegments):
     plt.show()
 
 
-
+def getDataFromJson(inputFile):
+    data = []
+    raw_data = json.loads(open(inputFile, 'r').read())
+    for i in range(len(raw_data)):
+        data.append(raw_data[i])
+    return data
```

Comment:
This is just copying the data, right? Do you need to copy it? If so, there are probably better ways to copy a Python array.